docs/content/gds/admin/configuration/bff.en.md

```diff
@@ -1,7 +1,172 @@
+
+List of environment variables:
+
+- `GDS_BFF_TESTNET_DIRECTORY_INSECURE`
+- `GDS_BFF_TESTNET_DIRECTORY_ENDPOINT`
+- `GDS_BFF_TESTNET_DIRECTORY_TIMEOUT`
+- `GDS_BFF_MAINNET_DIRECTORY_INSECURE`
+- `GDS_BFF_MAINNET_DIRECTORY_ENDPOINT`
+- `GDS_BFF_MAINNET_DIRECTORY_TIMEOUT`
+
+#### GDS Members API Configuration
+
+The BFF connects to the secure GDS Members API to give logged in users access to the complete directory including listing verified members and updating their member record. The configuration for the members API client is as follows:
+
+| EnvVar                 | Type     | Default | Description                                                         |
+|------------------------|----------|---------|---------------------------------------------------------------------|
+| MEMBERS_ENDPOINT       | string   |         | The endpoint (host:port) to connect to the members API on           |
+| MEMBERS_TIMEOUT        | duration | 10s     | The connection timeout for members API request and dial contexts    |
+| MEMBERS_MTLS_INSECURE  | bool     | false   | If false does not connect to the members API using mTLS             |
+| MEMBERS_MTLS_CERT_PATH | string   |         | The path to the mTLS client-side certs for members API auth         |
+| MEMBERS_MTLS_POOL_PATH | string   |         | The path to the mTLS public cert pools to accept server connections |
+
+List of environment variables:
+
+- `GDS_BFF_TESTNET_MEMBERS_ENDPOINT`
+- `GDS_BFF_TESTNET_MEMBERS_TIMEOUT`
+- `GDS_BFF_TESTNET_MEMBERS_MTLS_INSECURE`
+- `GDS_BFF_TESTNET_MEMBERS_MTLS_CERT_PATH`
+- `GDS_BFF_TESTNET_MEMBERS_MTLS_POOL_PATH`
+- `GDS_BFF_MAINNET_MEMBERS_ENDPOINT`
+- `GDS_BFF_MAINNET_MEMBERS_TIMEOUT`
+- `GDS_BFF_MAINNET_MEMBERS_MTLS_INSECURE`
+- `GDS_BFF_MAINNET_MEMBERS_MTLS_CERT_PATH`
+- `GDS_BFF_MAINNET_MEMBERS_MTLS_POOL_PATH`
+
+### Database
+
+The BFF makes use of a globally replicated key-value store for persistence of BFF-specific data structures such as organizations and announcements. By default it uses the TrtlDB for this, but for testing or smaller deployments it can use a local LevelDB database instead.
+
+Generally speaking, the BFF uses the same TrtlDB as the GDS MainNet instance, and we've ensured there are no namespace conflicts to prevent this. The BFF primary store can also be independent of the GDS stores if necessary.
+
+The primary database store is configured as follows:
+
+| EnvVar                       | Type   | Default | Description                                                                          |
+|------------------------------|--------|---------|--------------------------------------------------------------------------------------|
+| GDS_BFF_DATABASE_URL             | string |         | Required, the DSN to connect to the database on (see below for details)              |
+| GDS_BFF_DATABASE_REINDEX_ON_BOOT | bool   | false   | When the server starts, instead of loading indexes from disk, recreate and save them |
```

Comment:
```suggestion
| GDS_BFF_DATABASE_REINDEX_ON_BOOT | bool   | false   | When the server starts, instead of loading indexes from disk, recreate and save them. |
```